inference-engine/src/transformations/src/transformations/common_optimizations/interpolate_sequence_fusion.cpp

```diff
@@ -0,0 +1,196 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "itt.hpp"
+#include "transformations/common_optimizations/interpolate_sequence_fusion.hpp"
+
+#include <algorithm>
+#include <memory>
+#include <numeric>
+#include <tuple>
+#include <utility>
+#include <vector>
+
+#include <ngraph/opsets/opset8.hpp>
+#include <ngraph/rt_info.hpp>
+#include <ngraph/pattern/op/wrap_type.hpp>
+
+namespace {
+using namespace ngraph;
+
+bool compatible_attrs(const opset8::Interpolate::InterpolateAttrs& fst_attrs, const opset8::Interpolate::InterpolateAttrs& snd_attrs) {
+    return std::tie(fst_attrs.mode, fst_attrs.shape_calculation_mode, fst_attrs.pads_begin, fst_attrs.pads_end,
+                    fst_attrs.coordinate_transformation_mode, fst_attrs.nearest_mode, fst_attrs.antialias, fst_attrs.cube_coeff) ==
+           std::tie(snd_attrs.mode, snd_attrs.shape_calculation_mode, snd_attrs.pads_begin, snd_attrs.pads_end,
+                    snd_attrs.coordinate_transformation_mode, snd_attrs.nearest_mode, snd_attrs.antialias, snd_attrs.cube_coeff);
+}
```

Comment:
I think it's better to add operator== to InterpolateAttrs struct.

Reply:
Done.